### DISCUSSION TITLE
How to fork the extension gallery website

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#1259] [Q&A] [answered]
I need help getting my fork of the Extension Gallery website up and running.

**Comment by CubesterYT** [comment, marked as answer]
**NOTE: You will need Git and NodeJS installed on your PC**

Once you fork TurboWarp/extensions, open the folder where you want your fork located in your PC, open a command shell at the folder location, and use `git clone {Your Forked Repository Git Link}`. Once you do that, in the same shell, use `npm ci` to install the dependencies of the fork. Once all of that is done, use `npm start` to start the fork, and you'll be given the localhost link in the shell where the fork is currently running. Hope this helps!

**Reply by BlueDome77** [reply]
thx!!
